package.json

```diff
@@ -8,7 +8,9 @@
     "prebuild": "rm -rf dist",
     "build": "tsc",
     "semantic-release": "semantic-release",
-    "test": "jest src --coverage"
+    "test": "jest src --coverage",
+    "format:check": "eslint src && prettier src --check",
+    "format:fix": "eslint src --fix && prettier src --write"
```

Comment:
this won't cover github workflows / anything outside of `src`. Do you want to instead use `*.yaml|.yml|.js|.ts| ...`